Validation: update demo description (#1998)
DevExpress/devextreme-demos: JSDemos/Demos/Validation/Overview/description.md

> This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on a page with enabled data validation. In this demo, the editors are grouped in an HTML form.

Reviewer: ```suggestion
This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on a page with enabled data validation. In this particular demo, the editors are grouped in an HTML form.
```

> This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on the page with enabled data validation. In this demo, the editors are collected in an HTML form.

Reviewer 2: > In this demo, the editors are **collected** in an HTML form.

Could you clarify what do you mean by "collected" here? Grouped?

> a group of editors on **the** page with enabled data validation

On **any** page with enabled data validation. The only necessary condition is "enabled data validation", and it could any page where it is possible to enable data validation.

```suggestion
This demo shows how to implement a [default validation group](/Documentation/Guide/UI_Components/Common/UI_Widgets/Data_Validation/#Validate_Several_Editor_Values/Group_the_Editors) - a group of editors on a page with enabled data validation. In this demo, the editors are collected in an HTML form.
```

> To enable editor's data validation, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement the [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach several validation rules to one component. Each validation rule contains a rule type that must be one of the following:

Reviewer 2: > To enable editor's data validation

Mind the articles, pretty please...

> You can attach **several** validation rules to one component.

I believed we've left this stuff in the past, yet here we are... "Several" is a colloquialism compared to "multiple". Moreover, they essentially mean exactly the same. "Several" is not smaller than "multiple". Yet "multiple" suits formal speech, such as documentation.

> Each validation rule contains a rule type that must be one of the following:

This is a very strange sentence. An awfully complicated way to express something that simple... Did you wanted to say the following:

> Each validation rule contains one of the following rule types:

However, the list below just enumerates all available validation rule types. Therefore, this sentence is still very strange.

> The following list contains all available validation rule types:

```suggestion
To data validation for an editor, you need to declare the [Validator](/Documentation/ApiReference/UI_Components/dxValidator/) component and implement [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). You can attach multiple validation rules to one component. The following list contains all available validation rule types:
```

- [required](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/)    

> A validation rule that demands that the validated field has a value.

Reviewer: ```suggestion
A validation rule that requires the validated field to have a value.
```
Vlada, please make this correction in the other similar sentences. Thanks.

Author: Okay, I will do it

> A validation rule that demands that a validated field has a value.

Reviewer 2: the [currently] validated field, that is the field to which the examined rule is applied. You did it right in the line below. Just be more attentive.

```suggestion
A validation rule that demands that the validated field has a value.
```

- [email](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/EmailRule/)    

> A validation rule that demands that the validated field match the Email pattern.

Reviewer 2: the field match**es**, singular.

```suggestion
A validation rule that demands that the validated field matches the Email pattern.
```

- [async](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/)    
A custom validation rule used for server-side validation. Implement the [validationCallback](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/#validationCallback) function to validate the target value.

- [compare](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/)    

> A validation rule that demands that the validated editor's value is equal to a specified expression. For this rule type, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value that component compares to the validated value.

Reviewer: ```suggestion
A validation rule that requires the validated editor's value to equal 
the value of the specified expression. To apply this rule, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value against which this component compares the validated value.
```
Your paragraph didn't make sense to me, so I rewrote it. However, since I couldn't really understand what you wanted to say, we need to make sure that the technical meaning is correct in my rewrite.

Author: Yes, everything is correct, I guess I got confused in the last part of the sentence

> A validation rule that demands that a validated editor has a value that is equal to a specified expression. For this rule type, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value that component compares with a validated value.

Reviewer 2: Be simpler and more attentive.

```suggestion
A validation rule that demands that the validated editor's value is equal to a specified expression. For this rule type, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value that component compares to the validated value.
```

- [pattern](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/)    
A validation rule that demands that the validated field matches a specified [pattern](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/#pattern). 

- [stringLength](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/)   

> A validation rule that demands the target value length be within the range of specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max). This property accepts only string values.

Reviewer 2: ```suggestion
A validation rule that demands that the target value length is within the range between the specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max) values. This property accepts only string values.
```

> A validation rule that demands that the target value length is within the range between the specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max) values. This property accepts only string values.

Reviewer: ```suggestion
A validation rule that requires the target value length to fall within the range of the specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max) values. This property only accepts string values.
```
Please apply this edit to similar sentences in this doc.

Author: Okay

- [range](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/)  

> A validation rule that demands the target value length be within the range of specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/#max). This property accepts only date-time or numeric values.  

Reviewer 2: ```suggestion
A validation rule that demands the that target value length is within the range between the specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/#max) values. This property accepts only date-time and numeric values.  
```

- [numeric](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/NumericRule/)    
A validation rule that demands that the validated field has a numeric value.

- [custom](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CustomRule/)    
A rule with custom validation logic. Implement the [validationCallback](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/#validationCallback) function to validate the target value.

> No matter the type, all validation rules allow you to specify an error [message](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#message) for a component. If your application implements a validation group, you can use the [ValidationSummary](/Documentation/ApiReference/UI_Components/dxValidationSummary/) to display all validation errors in one place. In this demo, click the **Register** button to see a validation summary.

Reviewer 2: > No matter the type

This isn't the best way to start a paragraph in documentation... Moreover, I believe we can go by without it at all.

> you can use the [ValidationSummary](/Documentation/ApiReference/UI_Components/dxValidationSummary/) to display all validation errors in one place.

ValidationSummary - an API member used as ordinary words in a sentence... This isn't the way to do things properly.

```suggestion
All validation rule types allow you to specify an error [message](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#message) for a component. If your application implements a validation group, you can use a [validation summary](/Documentation/ApiReference/UI_Components/dxValidationSummary/) to display all validation errors in one place. In this demo, click the **Register** button to see a validation summary.
```

> The **Register** button's [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property is enabled. This setting tells the button to validate and submit the HTML form. In your application, you can also implement the button's [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) event handler and use the [validate()](/Documentation/ApiReference/UI_Components/dxValidationGroup/Methods/#validate) or [validateGroup()](/Documentation/ApiReference/Common/Utils/validationEngine/#validateGroup) methods to validate a group of editors.

Reviewer 2: ```suggestion
The **Register** button's [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property is enabled. As a result, a click on the button validates and submits the HTML form. In your application, you can also implement the button's [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) event handler and use the [validate()](/Documentation/ApiReference/UI_Components/dxValidationGroup/Methods/#validate) or [validateGroup()](/Documentation/ApiReference/Common/Utils/validationEngine/#validateGroup) method to validate a group of editors.
```
